README.md

```diff
@@ -1,5 +1,5 @@
 # hw01
-# **Story of my life**
+# **Story of My Life**
 
```

Comment:
In terms of other things to include, I don't see a "Report your process" section here or in the pull request comment section. What was hard/easy, some problems you solved, or helpful tutorials you read? Also, maybe include a little more about why you want to learn R and a little more detail on the old films you've been binging. I'm struggling to think of critical feedback since you included almost everything the assignment asked!

(Also, love the Star Trek meme).

Reply:
Thank you for the review (and for not being weirded out by the meme)! I did include a brief report, you can read it under the conversation section. Maybe it was too brief? Either way, thanks for bringing it up, I will try to write in detail for following projects.

Reply:
Sorry, not sure how I missed that! Glad it's included :)